packages/tools/pluggable-widgets-tools/configs/rollup-plugin-collect-native-deps.js

```diff
@@ -0,0 +1,123 @@
+import { basename, dirname, extname, join } from "path";
+import { copy, readJson, writeJson } from "fs-extra";
+import { promises } from "fs";
+import { rm } from "shelljs";
+
+export function collectNativeDependencies({ outputDir, externals, widgetName, shouldCopyNodeModules = true }) {
```

Comment:
- Why use object parameter iso regular parameters?
- Name of file and name of this function are slightly misaligned...

Reply:
Fixed